test/functional/mempool_limit.py

```diff
@@ -9,20 +9,17 @@
 
 class MempoolLimitTest(BitcoinTestFramework):
 
-    def setup_network(self):
-        self.nodes = []
-        self.nodes.append(start_node(0, self.options.tmpdir, ["-maxmempool=5", "-spendzeroconfchange=0"]))
-        self.sync_all()
-        self.relayfee = self.nodes[0].getnetworkinfo()['relayfee']
-
     def __init__(self):
         super().__init__()
         self.setup_clean_chain = True
         self.num_nodes = 1
+        self.extra_args = [["-maxmempool=5", "-spendzeroconfchange=0"]]
 
+    def run_test(self):
         self.txouts = gen_return_txouts()
```

Comment:
As below. I'd prefer `__init__()` to just be for setting up the test network topology.

However, these could be changed to be local variables instead of object variables.